queries/hcl/highlights.scm

```diff
@@ -54,18 +54,20 @@
 ] @conditional
 
 [
-  (string_lit)
-  (quoted_template)
-  (heredoc_template)
+  (quoted_template_start) ; "
+  (quoted_template_end); "
+  (template_literal) ; non-interpolation/directive content
 ] @string
 
-
 [
-  (heredoc_identifier)
-  (heredoc_start)
+  (heredoc_identifier) ; <<END
+  (heredoc_start) ; END
 ] @punctuation.delimiter
 
-(template_interpolation) @string.escape
+[
+  (template_interpolation_start) ; ${
+  (template_interpolation_end) ; }
+] @string.escape
```

Comment:
Could this be highlighted lik `${...}`in bash?

Reply:
In the bash highlights they are `@punctuation.bracket`. I think one could write something like

```
( template_interpolation [ (template_interpolation_start) (template_interpolation_end) ] @punctuation.bracket )
```

That would feel like the bash queries. Should i do it like this?

Reply:
I was more referring to the `@none` highlight in the beginning of the bash highlight to reset the string highlighting.

> That would feel like the bash queries. Should i do it like this?

I'd say yes. We should try to have bash, fish, cmake in sync. If you think the other solution is better you can create a PR changing it for all the languages.

Reply:
Ok, i changed it to `punctuation.bracket` and specialized the query to `template_interpolations`.

Reply:
It looks like this in my colorscheme now.

![Auswahl_028](https://user-images.githubusercontent.com/17451647/124292949-6c905200-db56-11eb-8f4b-2da0f08ca13a.png)

Reply:
Maybe string.escape fits better, but I would prefer to change it for all languages at once and this is better done in a separate PR.